File: arthur002/programaçao_basica/Questao09.java
import java.util.Scanner;

public class Questao09 {
    public static void main (String[] args) {
        double d;
        double ca;
        double cg;
        double ca1;
        double cg1;
        Scanner s;

        s = new Scanner(System.in);
        System.out.println("**Quantidade de Salgados**");

        System.out.print("Informe o total de dinheiro que possui:");
        d = s.nextDouble();
        System.out.print("Quanto custava um salgado?");
        ca = s.nextDouble();
        System.out.print("Quanto custa um salgado?");
        cg = s.nextDouble();
        
        ca1 = d/ca;
        cg1 = d/cg;

        System.out.println( );
        System.out.println("Antes com R$" + d + "voce podia comprar" + ca1 + "salgados");
        System.out.println("Agora com R$" + d + "voce pode comprar" + cg1 + "salgados");
    }
}

File: rute010/questao09.py
print("**Questao de Salgados**")

d = float(input("Informa o total de dinheiro que possui:"))
ca = float(input("Quanto custava um salgado?"))
cg = float(input("Quanto custa um salgado?"))

ca1 = (d)/(ca)
cg1 = (d)/(cg)
print( )
print("Antes com R$" + str(d) + "voce podia comprar" + str(ca1) + "salgados")
print("Agora com R$" + str(d) + "voce so  pode comprar" + str(cg1) + "salgados")
